microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Remote Debugging for WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge
+description: How to remotely debug WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge. 
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 02/23/2023
+---
+# Remote Debugging WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge
+
+Use `Remote Tools for Microsoft Edge` to enable [Microsoft Edge DevTools](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/landing/) debugging.
+
+
+<!-- ====================================================================== -->
+## Debugging Desktop WebView2 WinUI 2 (UWP) apps
+
+These steps can be used to attach `Microsoft Edge DevTools` remotely to a WebView2 WinUI 2 (UWP) app.
+Remote debugging is necessary as a current limitation to WebView2 WinUI 2 (UWP) apps is the built-in `Microsoft Edge DevTools` cannot be launched inside a store signed WebView2 WinUI 2 (UWP) app.
+
+1. Install [Edge Canary](https://www.microsoftedgeinsider.com/en-us/download/canary).
+
+1. In `Edge Canary` navigate to `edge://inspect` and verify the page is supported in your version of the browser.
+
+1. Enable `Developer Mode`:
+
+    Open **Windows Settings> Privacy & security > For developers**:
+
+    Switch `Developer Mode` to **On**.
+
+    Verify the switch is enabled:
+
+    ![Enable Developer Mode](./media/enable-developer-mode.png)
+
+1. Enable `Device Portal`:
+
+    Open **Windows Settings> Privacy & security > For developers**:
+
+    Switch `Device Portal` to **On**.
+
+    Click **Yes** to Install Windows Developer Mode package when prompted.
+
+    ![Install Windows Developer Mode package](./media/install-developer-mode-package.png)
+
+    Once the installation is complete, note the URL that will be used for the remote debugging connection. To find this URL, expand the Device Portal dropdown.  In this case, the Device Portal URL that will be used is `http://localhost:50080`.
+
+    ![Remote Debugging URL](./media/remote-debugging-url.png)
+
+1. Install [Remote Tools for Microsoft Edge](https://www.microsoft.com/store/productId/9P6CMFV44ZLT) from the Microsoft Store.
+
+1. Open the Device Portal URL you previously noted in `Edge Canary`. Verify this is the page that is loaded:
+
+    ![Open Device Portal](./media/open-device-portal.png)

```

Comment:
global within each numbered list: 1 or 2 spaces after "1. ", text must align w/ image tagging consistently in that list.